### PR TITLE
fix/unicode

### LINKED ISSUE #121: [BUG] unescape() is deprecated
# bug

Unicode-encoded strings in params get mojibake.

# code

https://github.com/dreamos82/chrome_parameters/blob/73ea3cf308e0883bd177edecf6ab511931cf500f/src/popup.js#L270

## fix

It would be a solution to fix this by using `decodeURIComponent` like this:

```js
function decodeQueryParam(p) {
  return decodeURIComponent(p.replace(/\+/g, ' '));
}

decodeQueryParam('search+query');
```

# ref

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/unescape

> Usually, `decodeURI` or `decodeURIComponent` are preferred over `unescape`.

> Note: Do not use `unescape` to decode URIs, use `decodeURI` instead.

### PR DESCRIPTION
**Pull request by Wintus**
fix #121

### CODE REVIEW

File: src/popup.js [hunk 1]
```diff
@@ -6,247 +6,248 @@
...
+    }
 
-		new_url = new_url + parameter_name;
+    new_url = new_url + parameter_name;
 
-		if(parameter_value){
-			new_url = new_url + "=" + escape(parameter_value);
-		}
+    if (parameter_value) {
+      new_url = new_url + "=" + encodeURIComponent(parameter_value);
```
Comment (Wintus) [inline src/popup.js]: here is the problem

**Comment by inuyasha82** [comment]
Hey @Wintus thank you for your PR, i'm happy to merge, but just one question first to double check you reindentend the file on purpose?

**Comment by Wintus** [comment]
> reindentend the file on purpose?

Yes, the reformat commit is intentional. And it obviously breaks the consistency of the format...

I'll revert the noisy changes. Sorry for bothering.

**Comment by dreamos82** [comment]
Thank you @Wintus  btw ther was no need to revert, it was fine. I was just to ask to make sure it was intentional. 

Btw i'll merge it thank you!

**Pull request merged**